src/API/Init.php

```diff
@@ -50,48 +52,17 @@ public function __construct() {
 	 */
 	public function rest_api_init() {
 		$controllers = array(
+			'Automattic\WooCommerce\Admin\API\Features',
 			'Automattic\WooCommerce\Admin\API\Notes',
 			'Automattic\WooCommerce\Admin\API\NoteActions',
-			'Automattic\WooCommerce\Admin\API\Coupons',
 			'Automattic\WooCommerce\Admin\API\Customers',
```

Comment:
The `Customers` endpoint is actually powered by the `Reports\Customers` controller, so it should probably be inside the conditional.

Reply:
👍🏽